admin/includes/languages/english.php

```diff
@@ -20,7 +20,7 @@
 define('HEADER_LOGO_IMAGE', 'logo.gif');
 
 // look in your $PATH_LOCALE/locale directory for available locales..
-$locales = array('en_US', 'en_US.utf8', 'en', 'English_United States.1252');
```

Comment:
why are we changing array() to [] ?

Reply:
I've been changing to the short form on files I modify, I understand it's preferred.

Reply:
^^^^ @zcwilt @drbyte please confirm and I will update the docs.

Reply:
definitely prefer the short form.